tools/build-asar.py

```diff
@@ -0,0 +1,14 @@
+import os
```

Comment:
IMO it would be preferable to reuse `npm_action` for this and put the `gn-asar.js` invocation in an npm script.

Reply:
@nornagon Thought about this, how about a new `node_action` that runs an arbitrary `.js` script.  Not too keen on polluting the `package.json.scripts` section just to make GN commands work